src/react/components/pages/editorPage/canvas.test.tsx

```diff
@@ -512,7 +512,7 @@ describe("Editor Canvas", () => {
                 ...cProps.selectedAsset,
                 regions: [copiedRegion],
             },
-        }).find(Canvas);
```

Comment:
how does the tests still pass with this gone / how does this not break tests before?

Reply:
The `wrapper` is the canvas and this isn't needed.  If you take a look at `createComponent()` the root element that it returns is the `Canvas` component itself.